capstone/src/main/webapp/multilinevisualization.js

```diff
@@ -132,11 +136,18 @@ function createMultilineVisualization() {
       .attr('font-weight', 'bold')
       .text('# of Page Views');
 
+  // Create random colors array, one per line
+  const colors = [];
+  for (let i = 0; i < data.restaurantData.length; i++) {
+    const rand = Math.random();
+    const color = 'hsl(" + rand*360 + ",100%,50%';
```

Comment:
Is this line valid? It looks like ' and " are being mixed possibly?

Reply:
It was invalid--thank you for double-checking that!! Should be fixed in the newest commit.